beacon-chain/powchain/service.go

```diff
@@ -181,6 +196,20 @@ func (w *Web3Service) Stop() error {
 func (w *Web3Service) ChainStartFeed() *event.Feed {
 	return w.chainStartFeed
 }
+func (w *Web3Service) saveToDb() {
+	powDepState := &pb.POWDepositState{}
+	if (w.depositTrie != nil && w.depositTrie != &trieutil.DepositTrie{}) {
+		(*powDepState).DepositTrie = w.depositTrie.ToProtoDepositTrie()
```

Comment:
Why do you reference the pointer of the pointer to powDepState? Shouldn't this just be `powDepState.DepositTrie = ...`